remo/voting/tasks.py

```diff
@@ -138,7 +141,7 @@ def create_rotm_poll():
                                     end__range=[start, end])
 
     if not now().date() > ROTM_NOMINATION_END_DATE or rotm_poll.exists():
```

Comment:
Why aren't you adding the waffle flag check here. Something like:

``
if not now().date() > ROMT_NOMINATION_END_DATE or rotm_poll.exists() or (waffle_check)
``